awx/main/models/ha.py

```diff
@@ -67,19 +68,28 @@ class InstanceLink(BaseModel):
     class States(models.TextChoices):
         ADDING = 'adding', _('Adding')
         ESTABLISHED = 'established', _('Established')
+        DISCONNECTED = 'disconnected', _('Disconnected')
         REMOVING = 'removing', _('Removing')
 
     link_state = models.CharField(
-        choices=States.choices, default=States.ESTABLISHED, max_length=16, help_text=_("Indicates the current life cycle stage of this peer link.")
+        choices=States.choices, default=States.ADDING, max_length=16, help_text=_("Indicates the current life cycle stage of this peer link.")
```

Comment:
I think this is a mistake, bound up in a misapprehension of what the modeling of the links is there to accomplish, along with several other things I see in this PR.  I'll try to write up my thoughts on this as a top-level comment.

Reply:
Jeff and I discussed and decided to keep ADDING as the default.